package.json

```diff
@@ -79,48 +79,48 @@
         "tailwind-merge": "^2.5.4",
         "tslib": "^2.8.1",
         "typedoc": "0.27.5",
-        "zone.js": "~0.14.10"
+        "zone.js": "0.15.0"
```

Comment:
don't we need patch changes? :) `~0.15.0`?

Reply:
Not sure yet, this change was suggested by `nx migrate`. Besides, we have a lock file that freezes the dependencies.

Reply:
True, with the lock file it does not matter

Reply:
True, with the lock file it does not matter